src/data/services/LmsApiService.js

```diff
@@ -235,12 +237,24 @@ class LmsApiService {
     return LmsApiService.apiClient().get(`${LmsApiService.lmsIntegrationUrl}/configs/?${queryParams.toString()}`);
   }
 
-  static fetchContentMetadataItemTransmission(uuid, channelCode, configId) {
-    return LmsApiService.apiClient().get(`${LmsApiService.lmsContentSyncStatusUrl}/${uuid}/${channelCode}/${configId}`);
+  static fetchContentMetadataItemTransmission(uuid, channelCode, configId, currentPage, options) {
+    let queryParams = '';
+    if (!_.isEmpty(options)) {
+      queryParams = new URLSearchParams(options);
+    }
+    const paramString = `?${currentPage ? `page=${currentPage}` : ''}&${queryParams.toString()}`;
+    const getSyncStatusUrl = `${LmsApiService.lmsContentSyncStatusUrl}/${uuid}/${channelCode}/${configId}${paramString}`;
```

Comment:
Good candidate to roll these into helper methods, since it's only the sync status url value that differs between the two fetchXMetadataItemTransmission methods.

Reply:
good idea